core/templates/domain/collection/guest-collection-progress.service.ts

```diff
@@ -77,8 +82,9 @@ export class GuestCollectionProgressService {
   }
 
   // This method corresponds to collection_domain.get_next_exploration_id.
+  // A null value will be returned if no explorationIds exist.
```

Comment:
or if we completed all explorations, right?

Reply:
Updated the comment